acvm/src/lib.rs

```diff
@@ -233,21 +235,25 @@ pub fn hash_constraint_system(cs: &Circuit) -> [u8; 32] {
 // This is set to match the previous functionality that we had
 // Where we could deduce what black box functions were supported
```

Comment:
should change mention of black box function, since we changed the name

Reply:
Done